defaults/main.yml

```diff
@@ -24,12 +24,12 @@ cleanup: false
 working_dir: "/tmp/hsm-prep-working-dir"
 
 ### Luna Parameters
-luna_minclient_src: "file:///opt/luna/Linux-Minimal-Client.tar.gz"
-luna_binaries_src: "file:///opt/luna/bin/"
-luna_server_cert_src: "file:///opt/luna/cert/server/cacert.pem"
+luna_minclient_src: "file:///opt/luna/LunaClient-Minimal-10.7.2-16.x86_64.tar"
+luna_minclient_version: "10.7.2-16"
```

Comment:
Do we plan to do anything with this parameter?  Otherwise, if this is just informational, it should probably go into the README instead.